src/store/keyboard/context.js

```diff
@@ -0,0 +1,40 @@
+import { createContext, useContext, useState } from 'react';
+
+const KeyboardContext = createContext();
+
+export const KeyboardProvider = ({ children }) => {
+  const [isOptionKeyPressed, setIsOptionKeyPressed] = useState(false);
+
+  const initialvalue = {
+    isOptionKeyPressed,
+  };
+
+  return (
+    <KeyboardContext.Provider value={initialvalue}>
+      <div
+        role="button"
+        tabIndex={0}
+        onKeyDown={(e) => {
+          if (e.key === 'Alt') {
+            setIsOptionKeyPressed(true);
+          }
+        }}
+        onKeyUp={(e) => {
+          if (e.key === 'Alt') {
+            setIsOptionKeyPressed(false);
+          }
```

Comment:
I am using `e.key == 'Alt` because e.altkey doesn't change to false on `onKeyPressUp`.

Reply:
> change to false

change to true

Reply:
edited. thanks